Readme.md

```diff
@@ -819,20 +819,14 @@ Enable test mode to push all jobs into a `jobs` array. Make assertions against
 the jobs in that array to ensure code under test is correctly enqueuing jobs.
 
 ```js
-queue = require('kue').createQueue();
+global.KUE_TEST_MODE = true;
```

Comment:
I don't like global.KUE_TEST_MODE stuff, Why weren't you sticked to the `queue.testMode.enter()` ?